modules/core/test/v2/unit/coins/dot.ts

```diff
@@ -80,27 +90,6 @@ describe('DOT:', function () {
         should.notEqual(e, null);
       }
     });
-
-    it('should verify sign params if the key array contains addresses', function () {
```

Comment:
not needed since key array is not actually part of the SignTransactionOptions in verifySignTransactionParams method.

Sample json object of what the params looks like:

```
{
  recipients: [
    {
      amount: '210000000',
      address: '5FodfGfp6zf9PZ1oKVs33H298zRwJCq2j5uEDEa9AjcYoNQC'
    }
  ],
  sequenceId: '0',
  type: 'transfer',
  walletPassphrase: 'secretpassphrase8u7y6t5r',
  wallet: { addressVersion: undefined },
  txPrebuild: {
    txHex: '0x9c040300a57220430d68b9055bb8540bd3b5519ae216b6f1316f8655a0e6c597edd29b9c026211320d680000aa23000008000000e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e325bacfc70deff6c82413730bf1ed3f7e77bdfdbbf607a912d79215ff4b37a0e',
    transaction: {
      id: '0x378b264d958ee9e135dbbc38e8643f08fba8c52081c1cc35ce1e6fcc3b81aee4',
      sender: '5GqykS8Cw2pZD3DxQ2FxMn6sKDrrVxghLH4BxxU5tk2sfz6V',
      referenceBlock: '0x325bacfc70deff6c82413730bf1ed3f7e77bdfdbbf607a912d79215ff4b37a0e',
      blockNumber: 9116160,
      genesisHash: '0xe143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e',
      nonce: 0,
      specVersion: 9130,
      transactionVersion: 8,
      eraPeriod: 16384,
      chainName: 'Westend',
      tip: 0,
      to: '5FodfGfp6zf9PZ1oKVs33H298zRwJCq2j5uEDEa9AjcYoNQC',
      amount: '210000000'
    },
    recipients: [
      {
        amount: '210000000',
        address: '5FodfGfp6zf9PZ1oKVs33H298zRwJCq2j5uEDEa9AjcYoNQC'
      }
    ],
    coin: 'tdot',
    walletId: '61de017f28c4fc0007e20707ac57974d',
    walletContractAddress: undefined
  },
  keychain: {
    id: '61de017f28c4fc0007e206fa32325c34',
    pub: 'd37879d79b6e58355d376192afce0bcd4d7ee740c07d19b9b5ee7ee336bc4f7d',
    source: 'user',
    encryptedPrv: '{"iv":"7oymsLg2EyXB7ZIhoqp1eg==","v":1,"iter":10000,"ks":256,"ts":64,"mode":"ccm","adata":"","cipher":"aes","salt":"2IbbodxIMNk=","ct":"LLtw46Tk3zpKweqeMFD5kjoHSDgW6UKO2KR6vtl7zT1hOuRRsaePySWwrlvx8pI0Uo7Ph6p2uhCGMVl+cmwN+UxbL2Xv4bB/"}',
    coinSpecific: {}
  },
  pubs: [
    'd37879d79b6e58355d376192afce0bcd4d7ee740c07d19b9b5ee7ee336bc4f7d'
  ],
  prv: 'xxxx'
}

```